### PR TITLE
[WIP] updates to Server metrics guide

### LINKED ISSUE #4348: Rework Monitoring and Metrics for Server
Slack Convo: https://circleci.slack.com/archives/C0KBNJGRH/p1586340997048200

### PR DESCRIPTION
**Pull request by rosieyohannan**
fixes #4348 

add this content: https://gist.github.com/makotom/f84d0db9078034a93e122ef92c469a2f

### CODE REVIEW

File: jekyll/_cci2/monitoring.adoc [hunk 1]
```diff
@@ -205,7 +269,114 @@ image::custom_metrics.png[Custom Metrics]
...
+    # Specify namespace for easier monitoring
+    namespace = 'my-circleci-server'
+
+    # Name of metrics key to record
+    namepass = ['mem']
+    # Name of metrics field to record; key and field are delimited by an underscore (_)
+    fieldpass = ['available_percent']
+```
 
 === Configuring Custom Metrics
```
Comment (rosieyohannan) [inline jekyll/_cci2/monitoring.adoc:381]: @makotom with all this new really great info you've given us, I'm not sure if we still need this section? Or maybe it just needs a better heading?

**Comment by teesloane** [comment]
This covers lots of ground in a domain I am unfamiliar with, but the writing is clear and concise!

File: jekyll/_cci2/monitoring.adoc [hunk 2]
```diff
@@ -12,15 +12,64 @@ This section includes information on metrics for monitoring your CircleCI Server
...
+* `\[[outputs.file]]` – Output configuration to emit metrics to stdout. All accepted metrics are configured to be shown in Telegraf docker logs. This is helpful for debugging your metrics configuration.
+* `\[[outputs.cloudwatch]]` – Output configuration to emit metrics to CloudWatch
+* `\[[outputs.datadog]]` – Output configuration to emit metrics to Datadog
+
+This configuration file is automatically generated by Replicated (the service used to manage and deploy CircleCI Server) and is fully managed by Replicated. If you wish to customize the standard configuration you will need to configure Replicated to **not** insert the blocks you want to change. 
+
+CAUTION: Do not attempt to directly modify the file. Any changes made in this way will be destroyed by Replicated upon certain events, such as service restarts. For example, if a customized `\[[inputs.statsd]]` block is added _without_ stopping automatic interpolation, you will encounter errors as Telegraf attempts to listen to `8125/UDP` twice, and the second attempt will fail with `EADDRINUSE`.
+
+In a standard configuration (with no metrics customization) the main config discussed above is for Telegraf.
```
Comment (makotom) [inline jekyll/_cci2/monitoring.adoc:60]: With this sentence I implied that there is nothing else but the main config to care about, if there is no metrics customization. I just want to make sure that the implication is conveyed with this wording. Thanks!

File: jekyll/_cci2/monitoring.adoc [hunk 3]
```diff
@@ -12,15 +12,64 @@ This section includes information on metrics for monitoring your CircleCI Server
...
+
+.Metrics
+image::metrics.png[Metrics]
+
+Metrics collection in CircleCI Server works as follows:
+
+* Each component of a Server installation sends metrics data to the telegraf container running on the Services machine.
+* Telegraf listens on port 8125/UDP and receives data from all components (inputs) and applies configured filters to determine whether data should be kept or dropped.
+* For some metric-types, Telegraf keeps metrics data inside and calculates statistical data (such as max, min, mean, stdev, sum) periodically.
+* Finally, Telegraf sends out data to stdout and configured data platforms (outputs), such as stdout, Datadog and/or AWS CloudWatch.
```
Comment (makotom) [inline jekyll/_cci2/monitoring.adoc:35]: "stdout" appears twice (I realized that this is a fault in my original draft 😇 )

How about like this:
```
Finally, Telegraf sends out data to configured sinks (outputs), such as stdout on the Services box, Datadog and/or AWS CloudWatch.
```

**Pull request merged**